mobile-browser-based-version/src/components/building_frames/containers/TrainingFrame.vue

```diff
@@ -21,15 +21,21 @@
         <div v-if="!training_manager.isTraining">
           <custom-button
             :center="true"
-            @click="training_manager.joinTraining(false)"
+            @click="training_manager.joinTraining(localScheme)"
           >
             Train Locally
           </custom-button>
           <custom-button
             :center="true"
-            @click="training_manager.joinTraining(true)"
+            @click="training_manager.joinTraining(decentralizedScheme)"
```

Comment:
@s314cy @dgengler6  maybe we could add:

joinFederatedTraining(), joinDecentralizedTraining(), joinLocalTraining(), into the training manager, that way we don't pass arguments and it's resolved in the training_manager?